api.ini

```diff
@@ -41,5 +41,4 @@ spooler-chdir = %d
 spooler-import = mtp_%n/tasks.py
 cron = 0 3 -1 -1 -1 %d/venv/bin/python %d/manage.py clean_up
 cron = -1 -1 -1 -1 -1 %d/venv/bin/python %d/manage.py run_scheduled_commands
-cron = 30 13 -1 -1 -1 %d/venv/bin/python %d/manage.py check_tokens
```

Comment:
I haven't checked out this branch and looked at the whole code on my machine (and still looking at this PR) but what is this `check_tokens` task and is it related to the performance platform?

Reply:
it's not related to the performance platform stuff, but i mention in 2cf2230ff53a2af0aa42bd4694516ac2ba9332a5 that the management command had already been removed in 98f920fef289cff2781c7ed81384f26524a30b09

Reply:
Oh sorry, I didn't look at the individual commits and I missed that explanation - makes sense 👌